components/lvgl_esp32_drivers/lvgl_tft/Kconfig

```diff
@@ -167,11 +174,16 @@ menu "LittlevGL (LVGL) TFT Display controller"
             select LVGL_TFT_DISPLAY_CONTROLLER_SH1107
             select LVGL_TFT_DISPLAY_PROTOCOL_SPI
             select LVGL_TFT_DISPLAY_MONOCHROME
-        config LVGL_TFT_DISPLAY_USER_CONTROLLER_SSD1306
-            bool "SSD1306"
-            select LVGL_TFT_DISPLAY_CONTROLLER_SSD1306
-            select LVGL_TFT_DISPLAY_PROTOCOL_I2C
-            select LVGL_TFT_DISPLAY_MONOCHROME
+	config LVGL_TFT_DISPLAY_USER_CONTROLLER_SSD1306
+		bool "SSD1306"
+			select LVGL_TFT_DISPLAY_CONTROLLER_SSD1306
+			select LVGL_TFT_DISPLAY_PROTOCOL_I2C
+			select LVGL_TFT_DISPLAY_MONOCHROME
+	config LVGL_TFT_DISPLAY_USER_CONTROLLER_FT81X
+		bool "FT81X"
+			select LVGL_TFT_DISPLAY_CONTROLLER_FT81X
+			select LVGL_TFT_DISPLAY_PROTOCOL_SPI
+			select LVGL_DISPLAY_ORIENTATION_LANDSCAPE
```

Comment:
Can you remove this line? `select LVGL_DISPLAY_ORIENTATION_LANDSCAPE`, that is the only warning I'm getting so far.

Reply:
Ok, I have updated my fork to removed that. Sorry about that, it was something that should have been removed.